assignments/solutions/rustlatin/src/lib_3.rs

```diff
@@ -0,0 +1,20 @@
+use std::str::Chars;
+
+const VOWELS: [char; 5] = ['a', 'e', 'i', 'o', 'u'];
+
+fn rustlatin(sentence: &str) -> Vec<char> {
+    let mut collection_of_words = Vec::new();
+    
+    for word in sentence.split(' ') {
+        let first_char = word.chars().next().unwrap();
+        collection_of_words.push(first_char);
+    };
+    words
```

Comment:
```suggestion
    collection_of_chars
```

Reply:
Working on this material without working CI to test all the examples is going to be increasingly hazardous.